src/modelo/Pikachu.java

```diff
@@ -2,10 +2,12 @@
 
 public class Pikachu extends Enemigo {
 
+    private double resistenciaMagica = 35; //Debil contra magia
```

Comment:
Again "...contra magia". What about creating some constants which express this intention?